docs/_posts/Mary-Sci/2023-08-04-legclf_parameter_efficient_prediction_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Legal Parameter Efficiency Prediction in Domain-Specific Documents
```

Comment:
Please change the title to `Legal Text Classification of Law Stack Exchange Questions`.